changelogs/unreleased/6224-drop-instance-dump-method.yml

```
@@ -0,0 +1,4 @@
+description: Remove the inmanta.execute.runtime.Instance.dump method as it doesn't seem to be used and is currently broken.
+issue-nr: 6224
+change-type: patch
+destination-branches: [master, iso6, iso5]
```

```suggestion
destination-branches: [master]
```
Let's restrict this to master, just in case.